src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+let App = () => {
```

Comment:
Suggest using `const` because the variable won't be redefined

Reply:
fixed